.circleci/config.yml

```diff
@@ -0,0 +1,39 @@
+version: 2.1
+
+orbs:
+  security: studion/security@2.0.0
+
+pr-filters: &pr-filters
+  branches:
+    ignore: master
+
+trunk-filters: &trunk-filters
+  branches:
+    only: master
+    
+jobs:
+  components-security-scan:
```

Comment:
Not a request for change, just a thought:
Would it be easier to read if we switched wording in the name of the job to:
`security-scan-components` and `security-scan-examples`

As it is now, it took me a few seconds to process that `examples-security-scan` is actually scanning examples, and is not an example of security scan for testing purposes. 😄

Reply:
This is now overhauled, please take a look.